presto-atop/src/main/java/io/prestosql/plugin/atop/AtopTable.java

```diff
@@ -14,6 +14,8 @@
 package io.prestosql.plugin.atop;
 
 import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.errorprone.annotations.Immutable;
```

Comment:
We already use `javax.annotation.concurrent.Immutable` in a number of places. Having two different annotations that mean the same thing is confusing. If the goal is to switch from JSR-305 to Error Prone annotations, then this would seem like a reasonable transition path, but Error Prone doesn't seem to have replacements for all of them.

If we're going to switch just `@Immutable`, we should add a checker to ban usage of the other one.

Reply:
I don't think we should switch from the standard `Immutable` annotation. Other tools like IntelliJ and Spotbugs understand the official ones, but may not understand the ones from errorprone. I'm surprised errorprone doesn't just use the standard ones. If anything, errorprone should switch to the standard ones, or add support for them.

Reply:
Let me switch to `javax.annotation.concurrent.Immutable`

Reply:
According to [Immutable](https://errorprone.info/bugpattern/Immutable), it lloks like `javax.annotation.concurrent.Immutable` is not a valid immutable annotation.

```
Other versions of the annotation, such as javax.annotation.concurrent.Immutable, are currently not enforced.
```